src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -21,6 +22,7 @@ public static void main(String[] args) {
 	}
 
 	@Bean // this bean runs a code block exactly once upon initialisation of the program
+	@Profile("!uninitialised")
```

Comment:
The code in the bean runs once at the start of the program. The point is that it initializes the system with fake users and orders.

But the purpose of the profile is that we don't want this fake data for some of our tests, namely OrderServiceTests and MatcherTests, because in those tests suits we want a blank market. i.e. we want the market to be *uninitialised*

So @Profile(!"unintialised") means that when the profile "uninitialised" is active, then this bean won't activate.

The profile name is not good to be frank, but I couldn't think of anything better.